src/print.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable*/
+export default function printMe() {
+  if (typeof console !== 'undefined') {
+    console.log('Some debug information');
```

Comment:
- [ ] I suggest you remove all `console.log` functions from your code after you're done and committing. This is for debugging and is to be removed after development